### PR TITLE
Portal and core fixups.

### PR DESCRIPTION
**Pull request by sjpadgett**
- remove unnecessary app controller and templates
- more portal routes cleanup
- fix portal home fixed sidebar
- replace alertMsg() translates with new xl()
- hopefully last dialog sizing fixes
- added back an overflow auto class to dialog.
Nation Notes
- fix NN text and LBF template defaults and loading timing
- add allow text templates for new encounters.(reason)
Globals
- add save button to bottom of form
- remove unneeded code.

### CODE REVIEW

**Comment by sjpadgett** [comment]
I'm getting really excited about this release. Thanks to everybody's hard work, will be one our best to date.
Also can't get over how well RTL works in portal so great job @tywrenn

**Comment by sjpadgett** [comment]
This has to get me a couple points:
image

I think i'm done with this one....

File: interface/super/edit_globals.php [hunk 1]
```diff
@@ -857,50 +862,6 @@ function validate_file() {
...
-            $('.row  .control-label').removeClass('col-sm-4 offset-sm-1');
-            $('.row  .oe-input').removeClass('col-sm-4');
-        }
-    }
-});
-$(function () {
-    $(window).trigger('resize'); // to avoid repeating code triggers above on page open
-});
-</script>
-<script>
```
Comment (bradymiller) [inline interface/super/edit_globals.php:903]: fare thee well large block of code :)

**Comment by bradymiller** [comment]
you get 3 points :)
code looks good to me

File: interface/forms/newpatient/common.php [hunk 2]
```diff
@@ -699,6 +699,11 @@ function setgroup(gid, name) {
         $('#sensitivity-tooltip').attr({"title": <?php echo xlj('If set as high will restrict visibility of encounter to users belonging to certain groups (AROs). By default - Physicians and Administrators'); ?>, "data-toggle": "tooltip", "data-placement": "bottom"}).tooltip();
         $('#onset-tooltip').attr({"title": <?php echo xlj('Hospital date needed for successful billing of hospital encounters'); ?>, "data-toggle": "tooltip", "data-placement": "bottom"}).tooltip();
     });
+
 </script>
+
+<?php if (!empty($GLOBALS['text_templates_enabled'])) { ?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['web_root'] ?>/library/js/CustomTemplateLoader.js"></script>
```
Comment (tywrenn) [inline interface/forms/newpatient/common.php:706]: No js type attributes pls

**Comment by sjpadgett** [comment]
@bradymiller 
Going to bed. Will you merge this for me after Travis. thanks

**Comment by bradymiller** [comment]
hi @sjpadgett , Sure thing. Bringing it in now. Thanks!
Gonna make it 4 points now since you fixed the js type thingy

**Pull request merged**